chia/cmds/coin_funcs.py

```diff
@@ -252,3 +252,12 @@ async def async_split(
         tx_id = transaction.name.hex()
         print(f"Transaction sent: {tx_id}")
         print(f"To get status, use command: chia wallet get_transaction -f {fingerprint} -tx 0x{tx_id}")
+        dust_threshold = config.get("xch_spam_amount", 1000000)  # min amount per coin in mojo
+        spam_filter_after_n_txs = config.get("spam_filter_after_n_txs", 200)  # how many txs to wait before filtering
+        if final_amount_per_coin < dust_threshold and wallet_type == WalletType.STANDARD_WALLET:
+            print(
+                f"WARNING: The amount per coin: {amount_per_coin} is less than the dust threshold: "
+                f"{dust_threshold / mojo_per_unit}. Some or all of the Coins "
+                f"{'will' if number_of_coins > spam_filter_after_n_txs else 'may'} not show up in your wallet unless "
+                f"you decrease the dust limit to below {final_amount_per_coin} mojos or disable it by setting it to 0."
+            )
```

Comment:
there's no test coverage for this block

Reply:
all the cli tests are going to follow my proof of concept pr.